test.sh

```diff
@@ -39,22 +39,21 @@ on_exit() {
 
 trap on_exit EXIT
 
-
 # run 'runtests' in the container
 # extra arguments are passed to the 'run' command (example: -e FOO=bar is added to the list of args)
 docoruntests() {
-    docker-compose -f test-compose.yml run --rm -e LOCAL_USER_ID=$(id -u) $@ odoo runtests
+    docker-compose -f test-compose.yml run --rm -e LOCAL_USER_ID=999 $@ odoo runtests
 }
 # run 'runmigration' in the container
 # extra arguments are passed to the 'run' command (example: -e FOO=bar is added to the list of args)
 docorunmigration() {
-    docker-compose -f test-compose.yml run --rm -e LOCAL_USER_ID=$(id -u) $@ odoo runmigration
+    docker-compose -f test-compose.yml run --rm -e LOCAL_USER_ID=999 $@ odoo runmigration
 }
 docodown() {
     docker-compose -f test-compose.yml down
 }
 docoruncmd() {
-    docker-compose -f test-compose.yml run --rm -e LOCAL_USER_ID=$(id -u) $@
+    docker-compose -f test-compose.yml run --rm -e LOCAL_USER_ID=999 $@
```

Comment:
It's used only for info in echo, in the entrypoint not really useful, it's just when you overwrite in the the dev image, you can see if it correspond to your local uid in case you want to mount local files